migrations/versions/rev-2023-06-22-11:53:16-a15294581397_.py

```diff
@@ -0,0 +1,27 @@
+"""empty message
+
+Revision ID: a15294581397
+Revises: 9b6873db28df
+Create Date: 2023-06-22 11:53:16.566102
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'a15294581397'
+down_revision = '9b6873db28df'
```

Comment:
That probably needs to be `8cf2327b952b` which was introduced in the previous PR